Source/Phone/Call/Call.swift

```diff
@@ -21,10 +21,10 @@
 import ObjectMapper
 
 /// Represents a Spark call.
-public class Call {
+open class Call {
```

Comment:
If it's not necessary for swift3.0 migration, I suggest that we don't change it.